producer/src/block/block.go

```diff
@@ -13,24 +13,37 @@ type Block struct {
 	PreviousHash   []byte
 	MerkleRootHash []byte
 	Data           [][]byte
+	DataLen        uint16
```

Comment:
Lets put `DataLen` before the `Data` field.

Reply:
Done